src/utils/Draft.js

```diff
@@ -79,7 +79,7 @@ function botPicks() {
     const ratedPicks = [];
     const unratedPicks = [];
     for (let cardIndex = 0; cardIndex < botPack.length; cardIndex++) {
-      if (draft.ratings[botPack[cardIndex].details.name]) {
+      if (draft.ratings && draft.ratings[botPack[cardIndex].details.name]) {
```

Comment:
This isn't related to the filed issue, but it caused some problems for me locally because the draft didn't have ratings.

Reply:
This happens b/c you don't have enough draft data in your local DB - and mongoose just saves the empty dict as null. This change is fine, just wanted to share why this is happening